app/controllers/users_controller.rb

```diff
@@ -48,4 +49,8 @@ def photo_galleries
     @galleries = @user.photo_galleries
   end
 
+  def latest_photo_comments
+    @latest_photo_comments = @user.latest_photo_comments.limit(10)
+  end
+
```

Comment:
On 12/23/2015 09:07 AM, eno wrote:
> In app/controllers/users_controller.rb
> <https://github.com/jensitus/ur-org/pull/131#discussion_r48327427>:
>
>> @@ -48,4 +49,8 @@ def photo_galleries
>>      @galleries = @user.photo_galleries
>>    end
>>
>> +  def latest_photo_comments
>> +    @latest_photo_comments = @user.latest_photo_comments.limit(10)
>> +  end
>> +
>
> Also I fail to see why this would be a controller method. Can you not
> just use "@user.late..(10)" in the views? Or else, would it make sense
> to write a helper to be used in the views?


I don't know really, in the view I have to iterate over these comments:
<% @latest_photo_comments.each do |comment | %>
   <%= comment %>
<% end %>

is it the better way to do it this way:
<% @user.latest_photo_comments.limit(10).each do |comment | %>
<%= comment %>
<% end %>

?